test/benchee/formatters/console_test.exs

```diff
@@ -73,4 +112,12 @@ defmodule Benchee.Formatters.ConsoleTest do
     [_, _, _, _comp_header, _reference, slower] = Console.format(jobs)
     assert String.last(slower) != "\n"
   end
+
+  defp assert_column_width(name, string, expected_width) do
+    n = Regex.escape name
+    regex = Regex.compile! "(#{n} +([0-9\.]+|ips))( |$)"
+    assert Regex.match? regex, string
+    assert expected_width == Regex.run(regex, string, capture: :all_but_first)
+      |> hd() |> String.length()
```

Comment:
I'd be pro on removing teh empty parentheses and to write the pipe calls underneath each other (probably even aligned under `Regex.run`